content/blog/2015-07_Meet Nukomeeter  Adam.md

```diff
@@ -0,0 +1,41 @@
+---
+created_at: 2015-07-02
+kind: article
+publish: true
+title: "Meet Nukomeeter: Adam"
+authors:
+- adam
+- kasia
+tags: 
+- interview
```

Comment:
@myviridis possible tags are ['business', 'programming', 'innovation', 'society', 'technology', 'web', 'meetup'] 

they are defined in https://github.com/nukomeet/nukomeet.com/blob/master/Rules